.github/workflows/code-quality.yml

```
@@ -0,0 +1,74 @@
+name: Code Quality Checks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+
+  lint: #-----------------------------------------------------------------------
+    name: Lint PHP files
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v2
+
+      - name: Set up PHP envirnoment
```

```suggestion
      - name: Set up PHP environment
```
(here and everywhere else)